_docs/company-overview.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Company Overview
+subtitle: Learn more about what OAK as a company is all about.
+author: chris
+tags: [architecture]
+---
+
+![oak_event_driven](../../assets/img/oak-event-driven.png)
+
+[__OAK (On-chain Autonomous Kernel) Network__](https://oak.tech) is an automation blockchain for Defi and recurring payments across protocols. The blockchain is built on Parity Substrate with an event-driven execution model. Different from Ethereum’s transaction-based model, where a transaction requires private key signing, OAK allows transactions to be triggered by event signals such as time, price, event triggers and smart contract state changes. 
```

Comment:
do we really want to limit this to Defi here? I get it's a selling point but could also turn off devs not with those use cases

Reply:
That's fair. I can do another rev of this.